libLanlGeoMag/Lgm_ComputeLstarVersusPA.c

```diff
@@ -199,8 +199,8 @@ void Lgm_ComputeLstarVersusPA( long int Date, double UTC, Lgm_Vector *u, int nAl
                         printf("\t\t%sUTC, LSimple     = %g %g%s\n\n\n", PreStr, UTC, LSimple, PostStr );
                     }
                     MagEphemInfo->Lstar[i] = ( LS_Flag >= 0 ) ? LstarInfo2->LS : LGM_FILL_VALUE;
-                    MagEphemInfo->I[i]  = LstarInfo2->I[0]; // I[0] is I for the FL that the sat is on.
-                    MagEphemInfo->K[i]  = LstarInfo2->I[0]*sqrt(MagEphemInfo->Bm[i]*1e-5); // Second invariant
+                    MagEphemInfo->I_data[i]  = LstarInfo2->I_data[0]; // I[0] is I for the FL that the sat is on.
```

Comment:
Comment needs to be updated for consistency

Reply:
Also done, with a similar debug message above updated as well